aesara/tensor/random/basic.py

```diff
@@ -73,18 +73,18 @@ def rng_fn(cls, rng, loc, scale, size):
 
 
 class GammaRV(RandomVariable):
-    name = "halfnormal"
+    name = "gamma"
     ndim_supp = 0
     ndims_params = [0, 0]
     dtype = "floatX"
     _print_name = ("Gamma", "\\operatorname{Gamma}")
 
     def __call__(self, shape, rate, size=None, **kwargs):
-        return super().__call__(shape, 1.0 / rate, size=size, **kwargs)
+        return super().__call__(shape, rate, size=size, **kwargs)
 
     @classmethod
-    def rng_fn(cls, rng, shape, scale, size):
-        return stats.gamma.rvs(shape, scale=scale, size=size, random_state=rng)
+    def rng_fn(cls, rng, shape, rate, size):
```

Comment:
Not sure about this, I though it makes sense for the `__call__` and `rng_fn` to treat the same inputs equally, but I actually don't know what is the purpose of `__call__` here. Happy to revert.